packages/admin/admin/src/mui/menu/Item.tsx

```diff
@@ -1,111 +1,21 @@
-                right: 0,
-                bottom: 0,
-                width: 2,
-            },
-            "& [class*='MuiListItemIcon-root']": {
-                color: colors.textLevel1,
-                minWidth: 28,
-            },
-            "& [class*='MuiListItemText-inset']": {
-                paddingLeft: 28,
-            },
-            "& [class*='MuiSvgIcon-root']": {
-                fontSize: 16,
-            },
-        },
-        level1: {
-            borderBottom: `1px solid ${theme.palette.grey[50]}`,
-            boxSizing: "border-box",
-            color: colors.textLevel1,
-            paddingLeft: 20,
-            paddingRight: 20,
-            paddingTop: 16,
-            paddingBottom: 16,
-            "&[class*='Mui-selected']": {
-                backgroundColor: theme.palette.grey[50],
-                color: theme.palette.primary.main,
-                "&:after": {
-                    backgroundColor: theme.palette.primary.main,
-                },
-                "& [class*='MuiListItemIcon-root']": {
-                    color: theme.palette.primary.main,
-                },
-            },
-            "& [class*='MuiListItemText-primary']": {
-                fontWeight: theme.typography.fontWeightMedium,
-                fontSize: 16,
-                lineHeight: "20px",
-            },
-        },
-        level2: {
-            color: colors.textLevel2,
-            paddingLeft: 20,
-            paddingRight: 20,
-            paddingTop: 10,
-            paddingBottom: 10,
-            "&:last-child": {
-                borderBottom: `1px solid ${theme.palette.grey[50]}`,
-                boxSizing: "border-box",
-            },
-            "&[class*='Mui-selected']": {
-                backgroundColor: theme.palette.primary.main,
-                color: "#fff",
-                "&:after": {
-                    backgroundColor: theme.palette.primary.dark,
-                },
-                "&:hover": {
-                    backgroundColor: theme.palette.primary.dark,
-                },
-                "& [class*='MuiListItemText-primary']": {
-                    fontWeight: theme.typography.fontWeightBold,
-                },
-            },
-            "& [class*='MuiListItemText-root']": {
-                margin: 0,
-            },
-            "& [class*='MuiListItemText-primary']": {
-                fontWeight: theme.typography.fontWeightRegular,
-                fontSize: 14,
-                lineHeight: "20px",
-            },
-        },
-        hasIcon: {},
-        hasSecondaryText: {},
-        hasSecondaryAction: {
-            paddingRight: 18,
-        },
-    });
+import { MenuItemClassKey, styles } from "./Item.styles";
 
 export interface MenuItemProps extends MenuLevel {
     primary: React.ReactNode;
     secondary?: React.ReactNode;
     icon?: React.ReactElement;
     secondaryAction?: React.ReactNode;
+    isCollapsibleOpen?: boolean;
+    hasSubitems?: boolean;
```

Comment:
Why do you need this two states? Which styling does this affect?

Reply:
I need it in the .styles file for the border

Reply:
![Uploading image.png…]()

Reply:
<img width="963" alt="image" src="https://github.com/vivid-planet/comet/assets/56400587/c64ba081-1f67-48d8-a448-02088c0c5eed">